Source/Particles/Filter/FilterFunctors.H

```diff
@@ -137,7 +137,7 @@ public:
     /** Store physical time. */
     amrex::Real m_t;
```

Comment:
...with this:
```suggestion
    amrex::ParticleReal m_t;
```

Is that possible?